### PR TITLE
type Annotation Action (Github Action).

### LINKED ISSUE #21: Fix the type annotation Action (Github Action)
Fix the following file `https://github.com/Karim-53/Compare-xAI/blob/main/.github/workflows/type_annotation.yml`

you can learn more about the bug for example here: `https://github.com/Karim-53/Compare-xAI/actions/runs/3860872224/jobs/6581483838`

this link could help `https://github.com/marketplace/actions/typilus-suggest-python-type-annotations`

### PR DESCRIPTION
**Pull request by Rishav1707**
## This PR closes the following issue:
closes #21

### CODE REVIEW

**Comment by Rishav1707** [comment]
@Karim-53 Please review my changes...

File: .github/workflows/type_annotation.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: Typilus Type Annotation Suggestions
```
Comment (Karim-53) [inline .github/workflows/type_annotation.yml:1]:
```suggestion
# Source: https://github.com/marketplace/actions/typilus-suggest-python-type-annotations
name: Typilus Type Annotation Suggestions
```

File: .github/workflows/type_annotation.yml [hunk 2]
```diff
@@ -14,9 +14,9 @@ jobs:
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so that typilus can access it.
     - uses: actions/checkout@v2
-    - uses: typilus/typilus-action@master
+    - uses: typilus/typilus-action@main
```
Comment (Karim-53) [inline .github/workflows/type_annotation.yml:17]:
```suggestion
    - uses: typilus/typilus-action@v0.9
```

File: .github/workflows/type_annotation.yml [hunk 3]
```diff
@@ -14,9 +14,9 @@ jobs:
...
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        # MODEL_PATH: path/to/model.pkl.gz   # Optional: provide the path of a custom model instead of the pre-trained model.
+        MODEL_PATH: path/to/model.pkl.gz   # Optional: provide the path of a custom model instead of the pre-trained model.
```
Comment (Karim-53) [inline .github/workflows/type_annotation.yml:20]:
```suggestion
```
There is no saved model in the repo for now. All are created via scripts for now

**Review by Karim-53 — COMMENTED** [review]
try this first

**Pull request merged**